docs/configuration/traces-config.md

```diff
@@ -92,6 +92,31 @@ remote_write:
       [ password: <secret> ]
       [ password_file: <string> ]
 
+    # Configures opentelemetry exporters to use the OpenTelemetry auth extension `oauth2clientauthextension`.
+    # Can not be used in combination with `basic_auth`.
+    # See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/oauth2clientauthextension/README.md
+    oauth2:
+    # Configures the TLS settings specific to the oauth2 client
```

Comment:
I think this should go before the `tls` block.

Reply:
17596c13